Support linked node modules

When I use a linked package, and I run `vsce package` it fails with 
```
Error: Command failed: npm list --production --parseable --depth=99999
npm ERR! invalid: <my_linked_package>
```

steps to reproduce in PowerShell:
```
git clone https://github.com/chriscamicas/vscode-abl.git
git clone https://github.com/chriscamicas/abl-tmlanguage.git
cd abl-tmlanguage
npm link
cd ..
cd vscode-abl
npm install
npm link abl-tmlanguage
npm run build
```

I'm running Windows 10 x64

--- a/src/npm.ts
+++ b/src/npm.ts
@@ -54,7 +54,7 @@ function checkNPM(cancellationToken?: CancellationToken): Promise<void> {
 
 function getNpmDependencies(cwd: string): Promise<string[]> {
 	return checkNPM()
-		.then(() => exec('npm list --production --parseable --depth=99999', { cwd, maxBuffer: 5000 * 1024 }))
+		.then(() => exec('npm list --production --parseable --depth=99999 --link=true', { cwd, maxBuffer: 5000 * 1024 }))
 		.then(({ stdout }) => stdout
 			.split(/[\r\n]/)
 			.filter(dir => path.isAbsolute(dir)));

